deepspeed/ops/adam/fused_adam.py

```diff
@@ -124,29 +154,41 @@ def step(self, closure=None, grads=None, output_params=None, scale=None, grad_no
                     # In order to keep backward compatibility for the existing checkpoints, we use group['state'] to initialize state['step'] if it exists.
                     state['step'] = group.get('step', 0)
                     # Exponential moving average of gradient values
-                    state['exp_avg'] = torch.zeros_like(p.data)
+                    state['exp_avg'] = torch.zeros_like(p.data).float()
```

Comment:
Are these changes from new apex? They seem redundant since `p` is expected to be `float` anyway.